usaspending_api/.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All these .idea files should not be committed. Add .idea/* to the .gitignore file and remove these from the PR